From face1e81b7b687b2aa6dff36f6552f4129a4fb10 Mon Sep 17 00:00:00 2001
From: Xiaoyu Yao <xyao@apache.org>
Date: Thu, 21 Jan 2016 13:51:58 -0800
Subject: [PATCH 1577/2748] HADOOP-12659. Incorrect usage of config parameters
 in token manager of KMS. Contributed by Mingliang
 Liu.

(cherry picked from commit f3427d3766d7101d0d1c37d6281918551d221ebe)
(cherry picked from commit 67f711edba9fcb0b24e01c149455bc846c868f14)

Change-Id: I17e41205a90e1f4c9e31f4cda1f1dc4789d71439
---
 .../delegation/ZKDelegationTokenSecretManager.java |    2 +-
 .../delegation/web/DelegationTokenManager.java     |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/ZKDelegationTokenSecretManager.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/ZKDelegationTokenSecretManager.java
index da0e6ad..88b81b0 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/ZKDelegationTokenSecretManager.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/ZKDelegationTokenSecretManager.java
@@ -141,7 +141,7 @@ public ZKDelegationTokenSecretManager(Configuration conf) {
         conf.getLong(DelegationTokenManager.MAX_LIFETIME,
             DelegationTokenManager.MAX_LIFETIME_DEFAULT) * 1000,
         conf.getLong(DelegationTokenManager.RENEW_INTERVAL,
-            DelegationTokenManager.RENEW_INTERVAL_DEFAULT * 1000),
+            DelegationTokenManager.RENEW_INTERVAL_DEFAULT) * 1000,
         conf.getLong(DelegationTokenManager.REMOVAL_SCAN_INTERVAL,
             DelegationTokenManager.REMOVAL_SCAN_INTERVAL_DEFAULT) * 1000);
     shutdownTimeout = conf.getLong(ZK_DTSM_ZK_SHUTDOWN_TIMEOUT,
diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/web/DelegationTokenManager.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/web/DelegationTokenManager.java
index 5d86249..221b02a 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/web/DelegationTokenManager.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/web/DelegationTokenManager.java
@@ -68,8 +68,8 @@ public DelegationTokenSecretManager(Configuration conf, Text tokenKind) {
       super(conf.getLong(UPDATE_INTERVAL, UPDATE_INTERVAL_DEFAULT) * 1000,
           conf.getLong(MAX_LIFETIME, MAX_LIFETIME_DEFAULT) * 1000,
           conf.getLong(RENEW_INTERVAL, RENEW_INTERVAL_DEFAULT) * 1000,
-          conf.getLong(REMOVAL_SCAN_INTERVAL,
-              REMOVAL_SCAN_INTERVAL_DEFAULT * 1000));
+          conf.getLong(REMOVAL_SCAN_INTERVAL, REMOVAL_SCAN_INTERVAL_DEFAULT)
+              * 1000);
       this.tokenKind = tokenKind;
     }
 
-- 
1.7.9.5

